tests/test_diffeq/test_wrappedscipysolver.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pytest
+import pytest_cases
+from scipy.integrate._ivp import base, rk
+from scipy.integrate._ivp.common import OdeSolution
+
+from probnum import diffeq, randvars
+from probnum.diffeq import odesolution, wrappedscipyodesolution, wrappedscipysolver
+
+
+def case_lorenz():
+    y0 = np.array([0.0, 1.0, 1.05])
+    ode = diffeq.lorenz([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+def case_logistic():
+    y0 = np.array([0.1])
+    ode = diffeq.logistic([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+def case_lotkavolterra():
+    y0 = np.array([0.1, 0.1])
+    ode = diffeq.lotkavolterra([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+@pytest.fixture
+def y():
+    return np.array([0.1])
+
+
+@pytest.fixture
+def times():
+    return [0.0, 1.0]
+
+
+@pytest.fixture
+def start_point():
+    return 0.5
+
+
+@pytest.fixture
+def stop_point():
+    return 0.6
+
+
+@pytest.fixture
+def dense_output():
+    return [base.DenseOutput(0, 1)]
+
+
+@pytest.fixture
+def lst():
+    return list([randvars.Constant(1)])
+
+
+@pytest_cases.parametrize_with_cases("testsolver,scipysolver", cases=".")
+def test_initialise(testsolver, scipysolver):
+    time, state = testsolver.initialise()
+    time_scipy = scipysolver.t
+    state_scipy = scipysolver.y
+    np.testing.assert_allclose(time, time_scipy, atol=1e-14, rtol=1e-14)
+    np.testing.assert_allclose(state.mean[0], state_scipy[0], atol=1e-14, rtol=1e-14)
+
+
+@pytest_cases.parametrize_with_cases("testsolver,scipysolver", cases=".")
+def test_step_execution(testsolver, scipysolver, start_point, stop_point, y):
+    scipy_y_new, f_new = rk.rk_step(
+        scipysolver.fun,
+        start_point,
+        y,
+        scipysolver.f,
+        stop_point - start_point,
+        scipysolver.A,
+        scipysolver.B,
+        scipysolver.C,
+        scipysolver.K,
+    )
+    scipy_error_estimation = scipysolver._estimate_error(
+        scipysolver.K, stop_point - start_point
+    )
```

Comment:
how do you feel about comparing to `RungeKutta.step()` here, instead of `rk_step`? This would be -- loosely speaking -- harder to pass, right?

Reply:
You're right, thanks for that hint!